react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -11,30 +11,29 @@
    - Create a separate branch for this point from "nextjs-ssr-pages-api" branch. Branch name: "nextjs-ssr-app-router-api".
    - You must use file-based routing provided by next.js (App Router)
    - [Migrate from Pages Api to App Router](https://nextjs.org/docs/pages/building-your-application/upgrading/app-router-migration).
-4. Remix Integration:
-   - Create a separate branch for this point from the previous branch task. Branch name: "remix-ssr".
-   - Add the Remix framework to your vite config
-   - [Update routing](https://remix.run/docs/en/main/file-conventions/routes)
-   - Migrate your application to ssr with Remix
+4. [Optional] React Router 7 Integration:
+   - Create a separate branch for this point from the previous branch task. Branch name: "react-router-ssr".
+   - [Update routing](https://reactrouter.com/start/framework/routing)
+   - Migrate your application to ssr with React Router 7: data fetching should be done via loaders on the server-side
```

Comment:
`application to ssr`
let's use uppercase SSR for consistency with the docs modules

Reply:
fixed